controller/main_controller.php

```diff
@@ -0,0 +1,62 @@
+<?php
+/**
+ *
+ * Collapsible Categories extension for the phpBB Forum Software package.
+ *
+ * @copyright (c) 2015 phpBB Limited <https://www.phpbb.com>
+ * @license GNU General Public License, version 2 (GPL-2.0)
+ *
+ */
+
+namespace phpbb\collapsiblecategories\controller;
+
+class controller implements main_interface
+{
+	/** @var \phpbb\collapsiblecategories\operator\operator */
+	protected $operator;
+
+	/** @var \phpbb\request\request */
+	protected $request;
+
+	/**
+	* Constructor
+	*
+	* @param \phpbb\collapsiblecategories\operator\operator     $operator           Collapsiblecategories Operator object
+	* @param \phpbb\request\request                             $request            Request object
+	* @access public
+	*/
```

Comment:
Less space between things in the docblock. Just needs 1 space, but still align things like they are. They just go too far to the right of the screen ;)